sotodlib/tod_ops/gapfill.py

```diff
@@ -380,3 +383,69 @@ def get_contaminated_ranges(good_flags, bad_flags):
                 rs.add_interval(int(i0), int(i1))
     return contam
 
+
+def fill_glitches(aman, nbuf=10, use_pca=False, modes=3, signal=None,
+                  glitch_flags=None, merge_filled=True,
+                  filled_name='gap_filled'):
+    """
+    Top level function for filling glitches assuming glitch flags exist.
+    Wraps the other functions in the ``tod_ops.gapfill`` module.
```

Comment:
"assuming glitch flags exist" -> "using pre-computed glitch flags provided by the caller."
"Top level function for filling glitches"  -> "Fill glitches in time-ordered data"

Mention that it uses polynomial filling, by default, but can use PCA optionally.

This function fills glitches, using either a polynomial or PCA-based approach.